### PR TITLE
Gui to change settings

### PR DESCRIPTION
**Pull request by albert-smit**
Adds a simple screen to change where the log goes too.

\> feature

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/semgr8/view/SettingsController.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    private CheckBox consoleCheckBox;
+      
+    /**
+     * Saves the settings, called when ok or apply is called.
+     */
+    @FXML
+    private void saveSettings() {
+        Logger.setLogToFile(fileCheckBox.isSelected());
+        Logger.setLogToSysOut(consoleCheckBox.isSelected());
+    }
```
Comment (kiliangrashoff) [inline src/main/java/nl/tudelft/ti2206/semgr8/view/SettingsController.java]: Missing linebreak

File: src/main/java/nl/tudelft/ti2206/semgr8/view/SettingsController.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    /**
+     * exits without saving, called when cancel or ok is called.
+     */
+    @FXML
+    private void exitSettings() {
+        stage.close();
+    }
```
Comment (kiliangrashoff) [inline src/main/java/nl/tudelft/ti2206/semgr8/view/SettingsController.java]: Missing linebreak
> layout

File: src/main/java/nl/tudelft/ti2206/semgr8/view/SettingsController.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+     * Called when ok button is pressed, saves the settings and exits after that.
+     */
+    @FXML
+    private void saveAndExitSettings() {
+        saveSettings();
+        exitSettings();
+    }
+
+    /**
+     * intializes the window and controller.
```
Comment (kiliangrashoff) [inline src/main/java/nl/tudelft/ti2206/semgr8/view/SettingsController.java]: Capital letter missing
> documentation

File: src/main/resources/gameLayout.fxml [hunk 4]
```diff
@@ -1,11 +1,22 @@
...
-<Pane fx:controller="nl.tudelft.ti2206.semgr8.view.GUIController"
+<?import javafx.scene.control.MenuBar?>
+<?import javafx.scene.control.Menu?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Tooltip?>
+<BorderPane fx:controller="nl.tudelft.ti2206.semgr8.view.GUIController"
       xmlns:fx="http://javafx.com/fxml">
-    <GridPane fx:id="boardGrid" vgap="1" hgap="1"/>
-</Pane>
+	<top>
```
Comment (kiliangrashoff) [inline src/main/resources/gameLayout.fxml]: Indentation in this section is a bit weird
> layout

**Comment by kiliangrashoff** [comment]
LGTM, just some minor style issues to be fixed.

**Comment by albert-smit** [comment]
Style issues fixed

\>layout

**Comment by kiliangrashoff** [comment]
LGTM, merged.

**Pull request merged**